src/app/modules/sgid/sgid.service.ts

```diff
@@ -195,11 +201,11 @@ export class SgidServiceClass {
    * @param rememberMe - determines how long the JWT is valid for
    */
   createJwt(
-    data: { 'myinfo.nric_number': string },
+    data: SGIDScopeToValue,
     rememberMe: boolean,
   ): Result<{ jwt: string; maxAge: number }, ApplicationError> {
-    const userName = data['myinfo.nric_number']
-    const payload = { userName, rememberMe }
+    const userName = data[SGID_MYINFO_NRIC_NUMBER_SCOPE]
+    const payload = { userName, data, rememberMe }
```

Comment:
Woops. I just realised I shouldn't even be storing this data in the JWT. Back to the drawing board!